packages/seal/src/client.ts

```diff
@@ -345,4 +345,17 @@ export class SealClient {
 			throw toMajorityError(errors);
 		}
 	}
+
```

Comment:
It'd be hard to use this API since `fetchKeys` returns when it has enough derived keys, not necessarily all keys or the one requested here.

some options:
- I believe that in all the use cases we have in mind, we should be able to decrypt also in the sdk, right? if so, maybe this should be an extension of `decrypt`, i.e., the caller can call `decrypt` and also get the used derived keys? for that we would need also the encryption though
- expose something similar to decrypt, e.g.,` getKemKeys(sessionKey, txBytes, threshold) ` which returns keys from cache or calls `fetchKeys`

Reply:
Yea, I see your point. The user could call `fetchKeys` with t = n before get keys, but it's indeed inconvenient.

Isn't the `getKemKeys` idea basically just returning the keys from `fetchKeys`? Or more precisely, writing a wrapper around `fetchKeys` that does this.